packages/core/src/services/apiKeys/destroy.ts

```diff
@@ -0,0 +1,14 @@
+import { eq } from 'drizzle-orm'
+
+import { ApiKey } from '../../browser'
+import { database } from '../../client'
+import { Result, Transaction } from '../../lib'
+import { apiKeys } from '../../schema'
+
+export async function destroyApiKey(apiKey: ApiKey, db = database) {
+  return Transaction.call(async (trx) => {
+    await trx.delete(apiKeys).where(eq(apiKeys.id, apiKey.id))
```

Comment:
You check in frontend. But in backend do we check if it's last API key?

Reply:
will add it a future pr